test.py

```diff
@@ -135,20 +135,24 @@ def test_destroy_app(self):
 
     def test_update_frames_inclusive(self):
         """Tests for the case where a new frame is added in the return of frames()."""
-        orgFramesSet = {wrapper.widget() for wrapper in self.qiwis._wrapperWidgets["app1"]}
+        orgFramesSet = {wrapperWidget.widget()
+                        for wrapperWidget in self.qiwis._wrapperWidgets["app1"]}
         newFramesSet = orgFramesSet | {QWidget()}
-        self.qiwis._apps["app1"].frames.return_value = tuple(newFramesSet)
+        self.qiwis._apps["app1"].frames.return_value = (("title", frame) for frame in newFramesSet)
         self.qiwis.updateFrames("app1")
-        finalFramesSet = {wrapper.widget() for wrapper in self.qiwis._wrapperWidgets["app1"]}
+        finalFramesSet = {wrapperWidget.widget()
+                          for wrapperWidget in self.qiwis._wrapperWidgets["app1"]}
```

Comment:
I think `wrapper` is enough, but you can change it if you want to.

Reply:
I also considered to use `wrapper`, but as it is a Python grammatical term, so I remained it.
However, you said okay, I will change it!